BLE_SupportedFeatures/mbed-os.lib

```diff
@@ -0,0 +1 @@
+https://github.com/ARMmbed/mbed-os/#26606218ad9d1ee1c8781aa73774fd7ea3a7658e

```

Comment:
```suggestion
https://github.com/ARMmbed/mbed-os/

```

Reply:
That is right, i forked this repo yesterday, made the PR and apparently this fetched the hash of a specific release

Reply:
That is my oversight, I will make a note in the readme about what branch to target with PRs.